src/Query.php

```diff
@@ -298,6 +299,25 @@ public function query($matcher)
     }
 
     /**
+     * Add an aggregation to the elastic query object
+     *
+     * @param  array|Elastica\Facet\AbstractAggregation $aggregation One or multiple facets
+     * @return $this
+     */
+    public function aggregation($aggregation)
```

Comment:
Would you mind changing this to `aggregate()`? I can add the test cases for you :)